components/engine_rocks/src/write_batch.rs

```diff
@@ -99,15 +99,17 @@ impl RocksWriteBatchVec {
 }
 
 impl engine_traits::WriteBatch for RocksWriteBatchVec {
-    fn write_opt(&mut self, opts: &WriteOptions) -> Result<()> {
+    fn write_opt(&mut self, opts: &WriteOptions) -> Result<u64> {
         let opt: RocksWriteOptions = opts.into();
         if self.support_write_batch_vec {
```

Comment:
submitted a PR for this and it will be merged soon https://github.com/tikv/rocksdb/pull/307, now we can just leave a comment here and resolve it after rocksdb is updated.